### DISCUSSION TITLE
Which Bitcoind wallets are used?

### DISCUSSION THREAD

**Discussion by indeed-a-genius** [lightningnetwork/lnd#9201] [Q&A] [answered]
I'm running a pruned Bitcoin core with multiple wallets.
I'd like to run LND backed by the same bitcoind node by providing LND RPC access to it. Will LND create separate a bitcoind wallet on first launch? And not touch any of my current wallets, including the default one? If so, what will be the name of the new wallet(s)?

**Comment by guggero** [comment, marked as answer]
`lnd` has its own on-chain wallet (using the [`btcwallet`](https://github.com/btcsuite/btcwallet) Golang library), so there is no wallet interaction with `bitcoind`. In fact, you can even use a `bitcoind` node that doesn't have wallet support compiled in.

The only thing I would take a look at is the prune value of your node. From experience, pruning too aggressively (e.g. less than 100 GB) can lead to sync issues, especially if your `bitcoind` node isn't well connected to other nodes (e.g. Tor only).
